docs/Other SDL Features/Creating an App Service/index.md

```diff
@@ -535,7 +535,7 @@ sdlManager.fileManager.upload(file: artwork) { [weak self] (success, bytesAvaila
 
 @![android,javaSE,javaEE]
 ```java
-final SdlArtwork weatherImage = new SdlArtwork("sun", FileType.GRAPHIC_PNG, R.drawable.sun, true);
+final SdlArtwork weatherImage = new SdlArtwork("sun", FileType.GRAPHIC_PNG, <#image#>, true);
```

Comment:
Should this be made more clear which constructor is used or what data type actually goes here? such as the following
<#image_resourceId#>
<#image_uri#>
<#image_byteArray#>